simplefs/simplefs_test.go

```diff
@@ -174,14 +175,40 @@ func TestList(t *testing.T) {
 	require.Len(t, listResult.Entries, 1, "Expected 1 directory entries in listing")
 	require.Equal(t, listResult.Entries[0].Name, "jdoe")
 
+	t.Log("List directory before it's created")
+	path1 := keybase1.NewPathWithKbfs(`/private/jdoe`)
+	opid, err = sfs.SimpleFSMakeOpid(ctx)
+	require.NoError(t, err)
+	err = sfs.SimpleFSList(ctx, keybase1.SimpleFSListArg{
+		OpID: opid,
+		Path: path1,
+	})
+	require.NoError(t, err)
+	checkPendingOp(
+		ctx, t, sfs, opid, keybase1.AsyncOps_LIST, path1, keybase1.Path{}, true)
+	err = sfs.SimpleFSWait(ctx, opid)
+	require.NoError(t, err)
+	listResult, err = sfs.SimpleFSReadList(ctx, opid)
+	require.NoError(t, err)
+	require.Len(t, listResult.Entries, 0,
+		"Expected 0 directory entries in listing")
```

Comment:
Do you think it's worthwhile to have a test for SimpleFSCopyRecursive too, to cover both the copy and the recursive path?

Reply:
Sure, easy enough.  Done!